ansible/roles/test/tasks/log_flood.yml

```diff
@@ -0,0 +1,62 @@
+# Pull CPU and memory, send a script to create syslogs to the DUT, wait 5 seconds, and make sure device is still within norms
+
+- name: set default value for number of syslog messages
+  set_fact:
+      qty_flood_messages: 10000
+  when: qty_flood_messages is not defined
+
+- name: pull CPU utilization via shell
+  # Explanation: Run top command with 2 iterations, 5sec delay. Discard the first iteration, then grap the CPU line from the second,
```

Comment:
typo grep

Reply:
updated the file, fixed that typo and the same one later in the file.